cmd/login.go

```diff
@@ -31,6 +31,7 @@ func Login() *cobra.Command {
 	token := ""
 	cmd := &cobra.Command{
 		Use:   "login [url]",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
how does the error look like in this case?

Reply:
<img width="595" alt="Captura de pantalla 2021-06-30 a las 17 23 14" src="https://user-images.githubusercontent.com/25170843/123988027-f2ca5e00-d9c7-11eb-902d-235167c4e108.png">